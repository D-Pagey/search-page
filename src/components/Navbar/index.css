.navbar {
  background-color: #fff;
  width: 100%;
}

@media (min-width: 1024px) {
  .navbar {
    width: 95%;
  }
}

.navbar-title {
  color: rgb(1, 39, 1);
  font-size: 2rem;
  margin: 0 0 0 15px;
}