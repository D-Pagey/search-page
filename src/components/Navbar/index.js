import React from 'react';
import styled from 'styled-components';

const NavbarWrapper = styled.nav`
  background-color: #fff;
  width: 100%;
  @media (min-width: 1024px) {
    width: 95%;
  }
`;

const NavbarTitle = styled.h1`
  color: rgb(1, 39, 1);
  font-size: 2rem;
  margin: 0 0 0 15px;
`;

export default function Navbar() {
  return (
    <NavbarWrapper>
      <NavbarTitle>DROVER</NavbarTitle>
    </NavbarWrapper>
  );
}
