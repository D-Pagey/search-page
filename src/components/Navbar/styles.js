import styled from 'styled-components';

export const NavbarWrapper = styled.nav`
  background-color: #fff;
  width: 100%;
  @media (min-width: 1024px) {
    width: 95%;
  }
`;

export const NavbarTitle = styled.h1`
  color: rgb(1, 39, 1);
  font-size: 2rem;
  margin: 0 0 0 15px;
`;
