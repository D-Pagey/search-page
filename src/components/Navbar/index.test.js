import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '.';

it('should render', () => {
  const wrapper = shallow(<Navbar />);
  expect(wrapper).toMatchSnapshot();
});
