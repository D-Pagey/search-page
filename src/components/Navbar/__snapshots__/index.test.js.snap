// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<nav
  className="navbar"
>
  <h1
    className="navbar-title"
  >
    DROVER
  </h1>
</nav>
`;
